### ISSUE TITLE
[ERROR] Streaming error (Server sends an incompatible content-type) - EC: 4003

### ISSUE THREAD

**Issue by shuxiao9058** [xteve-project/xTeVe#34]
```
Oct  4 09:39:47 raspberrypi xteve[18575]: #033[35m2019/10/04 09:39:47 [DEBUG] Request:               * * * * * * BEGIN HTTP(S) REQUEST * * * * * *
Oct  4 09:39:47 raspberrypi xteve[18575]: #033[0m#033[35m2019/10/04 09:39:47 [DEBUG] Method:                GET
Oct  4 09:39:47 raspberrypi xteve[18575]: #033[0m#033[35m2019/10/04 09:39:47 [DEBUG] Proto:                 HTTP/1.1
Oct  4 09:39:47 raspberrypi xteve[18575]: #033[0m#033[35m2019/10/04 09:39:47 [DEBUG] URL:                   http://223.110.240.221:6610/cntv/live1/HD-2500k-1080P-beijingstv/HD-2500k-1080P-beijingstv/1.m3u8?IASHttpSessionId=OTT2651820191004083915000073
Oct  4 09:39:47 raspberrypi xteve[18575]: #033[0m#033[35m2019/10/04 09:39:47 [DEBUG] Header:                connection: close
Oct  4 09:39:47 raspberrypi xteve[18575]: #033[0m#033[35m2019/10/04 09:39:47 [DEBUG] Header:                accept: */*
Oct  4 09:39:47 raspberrypi xteve[18575]: #033[0m#033[35m2019/10/04 09:39:47 [DEBUG] Header:                user-agent: xTeVe
Oct  4 09:39:47 raspberrypi xteve[18575]: #033[0m#033[35m2019/10/04 09:39:47 [DEBUG] Request:               * * * * * * END HTTP(S) REQUEST * * * * * *
Oct  4 09:39:47 raspberrypi xteve[18575]: #033[0m#033[35m2019/10/04 09:39:47 [DEBUG] Server URL:            http://223.110.240.221:6610
Oct  4 09:39:47 raspberrypi xteve[18575]: #033[0m#033[35m2019/10/04 09:39:47 [DEBUG] Temp Folder:           /tmp/xteve/2019-10-CI18-YWSA3I/MB6GD6K3UUZSX1C4H3Y9/3052cbe57398701b779ba6c9121c9d7b/
Oct  4 09:39:47 raspberrypi xteve[18575]: #033[0m#033[0m2019/10/04 09:39:47 [xTeVe] Streaming Status:      HTTP Response Status [200] OK
Oct  4 09:39:47 raspberrypi xteve[18575]: #033[0m#033[0m2019/10/04 09:39:47 [xTeVe] Content Type:          audio/x-mpegurl
Oct  4 09:39:47 raspberrypi xteve[18575]: #033[0m#033[0m2019/10/04 09:39:47 [xTeVe] Content Type:          audio/x-mpegurl
Oct  4 09:39:47 raspberrypi xteve[18575]: #033[0m#033[31m2019/10/04 09:39:47 [xTeVe] [ERROR] Streaming error (Server sends an incompatible content-type) - EC: 4003
Oct  4 09:39:47 raspberrypi xteve[18575]: #033[0m#033[0m2019/10/04 09:39:47 [xTeVe] Streaming Status:      Client has terminated the connection
Oct  4 09:39:47 raspberrypi xteve[18575]: #033[0m#033[0m2019/10/04 09:39:47 [xTeVe] Streaming Status:      Channel: 北京卫视 (Clients: 0)
```

**Comment by alturismo** [comment]
may try latest beta with ffmpeg and/or vlc remux option.

**Comment by shuxiao9058** [comment]
fixed this with #35
